source/rust_verify/example/invariants.rs

```diff
@@ -1,3 +1,5 @@
+// rust_verify/tests/example.rs deprecated-enhanced-typecheck
```

Comment:
This implies `expect-success` right?

Reply:
Yes, because `expect-success` is the default.